my_list.rb

```diff
@@ -0,0 +1,10 @@
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- [ ] When I try to run your app, I get import error. Kindly fix that. You need to import the MyEnumerable module into this class for the `include` part to work.

  ```
  require_relative './my_enumerable'
  ```

  ![image](https://user-images.githubusercontent.com/54848363/230655576-cc7699b0-748a-4c7f-a87a-737b5b092c32.png)